doc/user-guide/Getting-started.md

```diff
@@ -7,23 +7,53 @@ You can either compile everything from the source code or install binaries from
 
 Check out our tutorial [How to build MongooseIM from source code](How-to-build.md) for an introduction to compiling, building and testing MongooseIM.
 
-### Install from DMG
+### Install from package
+
+Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) page of Erlang Solution website, and download the version of MongooseIM you want. The following sections describe the installation process for different operating systems.
```

Comment:
Go to the [downloads](https://www.erlang-solutions.com/resources/download.html) section of the Erlang Solution website, and choose the version of MongooseIM you want. The following sections describe the installation process for different operating systems.